localstack/services/ram/provider.py

```diff
@@ -0,0 +1,10 @@
+import logging
+
+from localstack.aws.api.ram import RamApi
+from localstack.services.plugins import ServiceLifecycleHook
+
+LOG = logging.getLogger(__name__)
+
+
+class RamProvider(RamApi, ServiceLifecycleHook):
```

Comment:
nit: This provider should not implement a `ServiceLifecycleHook` here, it's not hooking into the lifecycle anywhere.

Reply:
Removed in 0d2dcd536 :+1: